extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java

```diff
@@ -22,6 +22,10 @@ public static WebClient createHttpClient(Vertx vertx, VaultBootstrapConfig vault
                 .setConnectTimeout((int) vaultBootstrapConfig.connectTimeout.toMillis())
                 .setIdleTimeout((int) vaultBootstrapConfig.readTimeout.getSeconds());
 
+        if (vaultBootstrapConfig.nonProxyHosts.isPresent()) {
```

Comment:
just a form thing: can be replaced with functional style expression: `vaultBootstrapConfig.nonProxyHosts.ifPresent(options::setNonProxyHosts);`

Reply:
Yeah, I think it's a matter of taste. Most of the time (i.e. when I don't have dozens of them), I prefer a simple if.